selfdrive/car/honda/carcontroller.py

```diff
@@ -140,12 +140,16 @@ def update(self, sendcan, enabled, CS, frame, final_gas, final_brake, final_stee
     tt = sec_since_boot()
     GAS_MAX = 1004
     BRAKE_MAX = 1024/4
-    STEER_MAX = 0xF00
+    if CS.crv:
+      STEER_MAX = 0xFFF/2
```

Comment:
Oh, yea, you should look in your CAN logs at the max value the stock system sent and not exceed that. It probably stopped steering and threw an error when you exceed MAX.